packages/ndk-multilib/postinst-header.in

```diff
@@ -0,0 +1,18 @@
+#!@TERMUX_PREFIX@/bin/sh
+
+pkg_format="@TERMUX_PACKAGE_FORMAT@"
+pkg_arch=unknown
+case "$pkg_format" in
+	debian ) pkg_arch=$(dpkg --print-architecture) ;;
+	pacman ) pkg_arch=$(sed -En 's/^\s*Architecture\s*=\s*(aarch64|arm|i686|x86_64)\s*$/\1/p' @TERMUX_PREFIX@/etc/pacman.conf) ;;
```

Comment:
I'm not familiar with pacman and I cannot figure out a more appropriate way to determine the native architecture. @Maxython What do you think?

Reply:
You can do that:
```bash
pkg_arch=$(pacman-conf Architecture)
```

Reply:
Thanks!